packages/main/src/SplitButton.ts

```diff
@@ -275,66 +292,119 @@ class SplitButton extends UI5Element {
 		};
 	}
 
+	/**
+	 * Function that makes sure the focus is properly managed.
+	 * @private
+	 */
+	_manageFocus(button?: Button | SplitButton) {
+		const buttons: Array<Button | SplitButton> = [this.textButton!, this.arrowButton!, this];
+
+		buttons.forEach(btn => {
+			if (btn) { btn.focused = btn === button; }
```

Comment:
Most probably it is redundant to check if  there is `btn` is present. From what we've tested there is always an array of three elements.